### DISCUSSION TITLE
Vuetify i Nuxt-18n

### DISCUSSION THREAD

**Discussion by IvanWala** [nuxt-modules/i18n#2706] [General]
Followin this example. Someone know how can we combine them for initilialize in nuxt and combine with vuetify? Specially for ltr rtl?

https://vuetifyjs.com/en/features/internationalization/#vue-i18n
https://vuetifyjs.com/en/getting-started/installation/#using-nuxt-3

**Comment by BobbieGoede** [comment]
I haven't tried vuetify with i18n myself, have you checked out https://github.com/userquin/vuetify-nuxt-module?

**Reply by IvanWala** [reply]
I will try this. Thanks.

**Comment by IvanWala** [comment]
Thanks @BobbieGoede seems to work.
